Update README.md (#3)
cloudinary-devs/react-photo-album: README.md

# Introduction

> This application showcases how to integrate the Cloudinary's React SDK. The sample project showcases the following functionality:

Reviewer: ```suggestion
This project demonstrates how to use the [Cloudinary React SDK](https://cloudinary.com/documentation/react_integration) to create a photo album app. 

The sample project showcases the following functionality:
```

> * Using the Upload Widget

Reviewer: ```suggestion
* Using the [Upload Widget](https://cloudinary.com/documentation/upload_widget) to upload images to your product environment
```

> * Uploading via API calls

Reviewer: ```suggestion
* Using the [Upload API](https://cloudinary.com/documentation/image_upload_api_reference) to upload images to your product environment
```

> * Displaying transformed images

Reviewer: ```suggestion
* [Transforming](https://cloudinary.com/documentation/image_transformations) and [displaying](https://cloudinary.com/documentation/react_image_transformations#plugins) images
```

# Setup

> Create an unsigned upload preset in your Cloudinary product environment and make note the name of the unsigned upload preset as well as your Cloudinary cloud name.

Reviewer: ```suggestion
[Create an unsigned upload preset](https://cloudinary.com/documentation/upload_presets#creating_and_managing_upload_presets) in your Cloudinary product environment and make note the name of the unsigned upload preset as well as your Cloudinary cloud name.
```

# Introduction
This project demonstrates how to use the [Cloudinary React SDK](https://cloudinary.com/documentation/react_integration) to create a photo album app. 

The sample project showcases the following functionality:

* Using the [Upload Widget](https://cloudinary.com/documentation/upload_widget) to upload images to your product environment

> * Using the [Upload API](https://cloudinary.com/documentation/image_upload_api_reference) to upload images to your product environment

Reviewer: ```suggestion
* Using the [REST API](https://cloudinary.com/documentation/client_side_uploading#direct_call_to_the_api) to upload images to your product environment
```

* [Transforming](https://cloudinary.com/documentation/image_transformations) and [displaying](https://cloudinary.com/documentation/react_image_transformations#plugins) images

# Setup

> [Create an unsigned upload preset](https://cloudinary.com/documentation/upload_presets#creating_and_managing_upload_presets) in your Cloudinary product environment and make note the name of the unsigned upload preset as well as your Cloudinary cloud name.

Reviewer: ```suggestion
[Create an unsigned upload preset](https://cloudinary.com/documentation/upload_presets#creating_and_managing_upload_presets) in your Cloudinary product environment and make a note of the name of the unsigned upload preset as well as your Cloudinary cloud name.
```

# Run

> From your terminal run `npm run dev` and follow the instructions to see this sample project.

Reviewer: ```suggestion
From your terminal, in the **react-pa** folder, run: 

* `npm i`
* `npm run dev` 

Then, click the localhost link to open the app in your browser.

Try uploading images using each of the upload tabs, then see your images displayed in the Photo Album tab.

# Explore

* Take a look at the source code to understand how it works.  
* Discover more in the [Cloudinary Docs](https://cloudinary.com/documentation).
* Ask for help in our [Community Forum](https://community.cloudinary.com/), in [Discord](https://discord.gg/Cloudinary), or [raise a support request](https://support.cloudinary.com/hc/en-us/requests/new).

```

# Run

> From your terminal, in the **react-pa** folder, run: 

Author: if they clone the repository the folder name will be react-photo-album so we should either state that or tell them to use the folder name they chose

Reviewer: Addressed in a suggestion further down.

> From your terminal, in the **react-pa** folder, run: 

Reviewer: ```suggestion
From your terminal, in the root folder of your project, run: 
```

* `npm i`
* `npm run dev` 

> Then, click the localhost link to open the app in your browser.

Author: "or open your browser and navigate to localhost:3000 or to the port displayed in the terminal"

Reviewer: Addressed in a suggestion further down.

> Then, click the localhost link to open the app in your browser.

Reviewer: ```suggestion
Then, click the localhost link to open the app in your browser, or open a browser and navigate to localhost:3000 or to the port displayed in the terminal.
```

Try uploading images using each of the upload tabs, then see your images displayed in the Photo Album tab.

# Explore

* Take a look at the source code to understand how it works.  

> * Discover more in the [Cloudinary Docs](https://cloudinary.com/documentation).

Reviewer: ```suggestion
* Discover more features in the [Cloudinary Docs](https://cloudinary.com/documentation).
```

* Ask for help in our [Community Forum](https://community.cloudinary.com/), in [Discord](https://discord.gg/Cloudinary), or [raise a support request](https://support.cloudinary.com/hc/en-us/requests/new).

> 

Reviewer: ```suggestion

Feel free to fork this repo as a starting point for your own React app, contribute to it, or star it if you like it!
```
